test/test_fx.py

```diff
@@ -65,6 +66,11 @@ def forward(self, x):
 def a_non_torch_leaf(a, b):
     return a + b
 
+# used in test_pytree. It's all the way out here because picking a GraphModule
```

Comment:
pickling?

Reply:
Yes, should be pickling, thanks for the catch